docs/sphinx/dev-guide/index.rst

```diff
@@ -1,14 +1 @@
-Pulp Developer Guide
-=======================
-
-.. toctree::
-   :maxdepth: 2
-
-   contributing/index
-   architecture
-   conventions/index
-   policies/index
-   newtypesupport/index
-   integration/index
-   glossary
-   troubleshooting
+The latest documentation has been moved to `here <http://pulp.readthedocs.org/en/latest/>`_. 
```

Comment:
Also it can just end in / instead of /en/latest/. RTD automaticaly redirects to /latest/ and selects the language based on the client so we should specify english.